app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -307,6 +307,8 @@ Watch our video interviews with non-UK citizens who've successfully relocated to
 
 $why-train-to-teach$
 
+[Read the transcript for the 'Why train to teach in England?' video](/non-uk-teachers/why-train-to-teach-transcript)
+
 [Join the Aspiring Teachers Forum on Facebook](https://www.facebook.com/groups/1357146377672255/).
```

Comment:
These two links (Join the forum and Read interviews) don't really make sense under the transcript. I would suggest moving them under `Understand the English education system` but need to check this with @laurahermionetennant92. If not, they need a separate heading above them?

Reply:
@gemmadallmandfe @caitlin-reidy thanks for the transcript – looks great. Yes please if you could move them under that heading, I think that would make sense.

Reply:
@laurahermionetennant92 can you check you're happy with them in the bulletpointed list now please?